File: Lab9_1/src/main/java/StripesSchema.java
public class StripesSchema {
    private final double[][] A;
    private final double[][] B;

    private double[][] C;

    private final int threadsNumb;

    public StripesSchema(double[][] A, double[][] B, int threadsNumb) {
        this.A = A;
        this.B = B;

        this.threadsNumb = threadsNumb;
    }

    public void calculateProduct() {

    }
}

File: Lab9_3/Lab9_3/StripesSchema.cpp
#include "StripesSchema.h"

StripesSchema::StripesSchema(double** A, double** B, int rows, int columns, int threadsNumber) :A(A), B(B), rows(rows), columns(columns), threadsNumber(threadsNumber)
{
	C = new double* [rows];
	for (int i = 0; i < rows; i++) {
		C[i] = new double[columns];
	}
}

double** StripesSchema::calculateProduct()
{
	int taskSize = rows / threadsNumber;
	/*
	for (int i = 0; i < threadsNumber; i++) {
		tbb::parallel_for(tbb::blocked_range<int>(0, threadsNumber), [&](tbb::blocked_range<int> r) {

			}

	}
	*/

	return nullptr;
}

void StripesSchema::compute(int rowStart, int rowEnd, int columnStart, int columnEnd)
{
}

double StripesSchema::calculateEntry(int i, int j)
{
	return 0.0;
}

